bouncy/distribution.yaml

```diff
@@ -383,6 +383,17 @@ repositories:
       url: https://github.com/eProsima/Fast-RTPS.git
       version: master
     status: developed
+  gazebo_ros_pkgs:
+    doc:
+      type: git
+      url: https://github.com/ros-simulation/gazebo_ros_pkgs.git
+      version: ros2
+    source:
+      test_pull_requests: true
```

Comment:
this line should be removed for now (as per https://github.com/ros2/rosdistro/pull/258#issuecomment-411182330)

Reply:
> Currently we don't support having PR testing for repository that are shared between the ROS1 and ROS 2 farm.

Why is that the case? I don't see a problem with a PR job if the devel job is supposed to work.

We currently can't build packages which have ROS 1 as well as ROS 2 dependencies - but that applies to all job types.

Reply:
It was the conclusion from https://github.com/ros2/rosdistro/pull/229#issuecomment-403584192 and follow-up meeting.

It was decided to not allocate time for testing this so we currently don't know how various jenkins instances managing webhooks on the org will behave. So the conclusion from the meeting was that we don't enable PR testing on repos used by both farms until we allocate time to test it and convince ourselves that it works without side effect

Reply:
We just talked about this in the ROS2 meeting. We were worried about conflicts near time of release. But this is important and we think it will likely work. And if it doesn't we should debug it now.